### ISSUE TITLE
Performance test - Object closed error 

### ISSUE THREAD

**Issue by rr83019** [librecaptcha/lc-core#42]
This is the most common type error occurring on both `/captcha` and `/answer` endpoint 
```
<Response [500]>
<!DOCTYPE html>
<html>
<head><title>500 Internal Server Error</title></head>
<body><h1>500 Internal Server Error</h1>
<p>Error processing request: The object is already closed [90007-197]</p>
</body></html>

**Comment by hrj** [comment]
Does this only occur during high load?

**Comment by hrj** [comment]
Does running the `scripts/simpleTest.py` work for you?

**Comment by rr83019** [comment]
> Does this only occur during high load?
No, it occurs at normal loads too. I've been testing for 1000 concurrent users spawning at 10 users/second.

**Comment by rr83019** [comment]
Stack trace:
```
[info] Starting server on port:8888
[info] org.h2.jdbc.JdbcSQLException: The object is already closed [90007-197]
[info]  at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
[info]  at org.h2.message.DbException.get(DbException.java:179)
[info]  at org.h2.message.DbException.get(DbException.java:155)
[info]  at org.h2.message.DbException.get(DbException.java:144)
[info] 	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:3202)
[info] 	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3115)
[info] 	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3247)
[info] 	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:310)
[info] 	at lc.Captcha.getChallenge(Main.scala:97)
[info] 	at lc.Server.$anonfun$new$1(Server.scala:75)
[info] 	at lc.HTTPServer.serve(Unknown Source)
[info] 	at lc.HTTPServer.handleMethod(Unknown Source)
[info] 	at lc.HTTPServer.handleTransaction(Unknown Source)
[info] 	at lc.HTTPServer.handleConnection(Unknown Source)
[info] 	at lc.HTTPServer$SocketHandlerThread$1.run(Unknown Source)
[info] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[info] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[info] 	at java.base/java.lang.Thread.run(Thread.java:834)

**Comment by rr83019** [comment]
> Does running the `scripts/simpleTest.py` work for you?
> 
> Note, the test doesn't attempt to solve the captcha correctly. So it will print a failure message at the end. But there should be no exceptions.

When I execute the script my cursor changes and when I click the cursor the following error occurs:
```
import-im6.q16: attempt to perform an operation not allowed by the security policy `PS' @ error/constitute.c/IsCoderAuthorized/408.

**Comment by hrj** [comment]
Closed via 62b3a098bdbe71ecd33194677e58801195b73682
